### ISSUE TITLE
--comm-as-payee (-x) is unrecognized (BZ#513)

### ISSUE THREAD

**Issue by tbm** [ledger/ledger#1585]
Note: the issue was created automatically with bugzilla2github

Bugzilla bug ID: BZ#513
From: John Wiegley (@jwiegley)

**Comment by jwiegley** [comment]
I do not believe that this is common enough that it needs its own option instead of just using `--payee=commodity`. Happy to hear counter-arguments.
